service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/SupportedContentTypes.java

```diff
@@ -0,0 +1,45 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.util.CollectionUtils;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import lombok.AllArgsConstructor;
+import lombok.SneakyThrows;
+import uk.nhs.adaptors.gp2gp.ehr.exception.EhrMapperException;
+
+@Component
+@AllArgsConstructor(onConstructor = @__(@Autowired))
+public class SupportedContentTypes {
+    private static final String SUPPORTED_CONTENT_TYPES_PATH = "supported-content-types.txt";
+    private static final List<String> SUPPORTED_CONTENT_TYPES = loadSupportedContentTypes();
+
+    public boolean isContentTypeSupported(String attachmentContentType) {
+        return !CollectionUtils.isEmpty(SUPPORTED_CONTENT_TYPES) && SUPPORTED_CONTENT_TYPES.contains(attachmentContentType);
```

Comment:
If there was a problem reading the file an `ExceptionInInitializerError` would be thrown and I think the service would fail to start. So, we shouldn't need to check for null
```suggestion
        return SUPPORTED_CONTENT_TYPES.contains(attachmentContentType);
```